### DISCUSSION TITLE
Βαθμολόγηση για αυτούς που δεν ειναι Κέρκυρα

### DISCUSSION THREAD

**Discussion by AlexandrosV20** [courses-ionio/hci#1577] [Announcements]
Καλησπέρα. Για τους επι πτυχίω φοιτητές που δεν βρίσκονται στην Κέρκυρα και δεν είναι εφικτό να παρακολουθήσουν το μάθημα δια ζώσης, πώς μπορούν να περάσουν την βάση π.χ στο εβδομαδιαίο quiz ή στην συμμετοχη στην τάξη. Επίσης αν κατάλαβα καλά οι παρουσιάσεις των παραδοτέων θα γίνεται ηλεκτρονικά για όσους δεν μπορούν ;

**Comment by mkamgit** [comment]
[Επι πτυχίω και φυσική παρουσία #1519](https://github.com/courses-ionio/hci/discussions/1519)

**Comment by AlexandrosV20** [comment]
Καλησπέρα,
Άρα αυτοι που δεν είναι Κέρκυρα, έχουν την δυνατότητα να παρουσιάσουν 
εξαποστάσεως αυτες τις 2-3 φορές ; Να ξέρετε είναι και το οικονομικό 
στην μέση για υτο και επιμένω να ρωτάω.

Στις 21-10-2021 09:30, Manousos Kamilakis έγραψε:
> Επι πτυχίω και φυσική παρουσία #1519 [1]
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub [2], or unsubscribe
> [3].
> Triage notifications on the go with GitHub Mobile for iOS [4] or
> Android [5].
> 
> Links:
> ------
> [1] https://github.com/courses-ionio/hci/discussions/1519
> [2] 
> https://github.com/courses-ionio/hci/discussions/1577#discussioncomment-1512461
> [3] 
> https://github.com/notifications/unsubscribe-auth/AEJCTCGAVTROXH3VCMKKZALUH6XQZANCNFSM5GHZVKEQ
> [4] 
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&amp;mt=8&amp;pt=524675
> [5] 
> https://play.google.com/store/apps/details?id=com.github.android&amp;referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub

-- 
με εκτίμηση,

Βυθούλκας Αλέξανδρος
Φοιτητής
Α.Μ Π2013021

ΔΗΛΩΣΗ ΕΜΠΙΣΤΕΥΤΙΚΟΤΗΤΑΣ / ΑΠΟΠΟΙΗΣΗ ΕΥΘΥΝΗΣ

Αυτό το μήνυμα περιέχει εμπιστευτικές πληροφορίες και η χρήση τους 
επιτρέπεται μόνον από τον αναφερόμενο παραλήπτη. Σε περίπτωση που 
περιέλθει σε σας από λάθος, παρακαλούμε να το διαγράψετε άμεσα από το 
σύστημά σας ή να το καταστρέψετε και να ειδοποιήσετε τον αποστολέα. Η 
αντιγραφή, χρήση ή κοινοποίηση σε τρίτους του μηνύματος αυτού από μη 
προορισμένο παραλήπτη, αντίκειται στο νόμο. Οι επικοινωνίες μέσω του 
Διαδικτύου δεν είναι ασφαλείς και για τον λόγο αυτό το Ιόνιο 
Πανεπιστήμιο δεν αποδέχεται νομική ευθύνη για τα περιεχόμενα του 
παρόντος μηνύματος και για οποιαδήποτε ζημιά προκληθεί από ιούς που 
είναι δυνατόν να εισαγάγει.

CONFIDENTIALITY WARNING / DISCLAIMER

The information in this email is confidential and is intended solely for 
the addressee(s). If you have received this transmission in error, and 
you are not an intended recipient,please delete it andnotify the sender. 
Be aware that if you are not the intended recipient any disclosure, 
copying, distribution or use of this transmission or its contents is 
prohibited. Internet communications are not secure and therefore Ionian 
Universitydoes not accept legal responsibility for the contents of this 
message and for any damage whatsoever that is caused by viruses being 
passed.»

**Reply by p14kairas** [reply]
Aς ελπίσουμε οτι θα γίνουν γιατί υπάρχουν πολλοί απο μακριά που δεν  μπορούν να πάνε Κέρκυρα αυθημερόν για μια παρουσίαση 10 λεπτών
